charts/cloudnative-pg/values.yaml

```diff
@@ -66,6 +66,14 @@ config:
 # -- Additinal arguments to be added to the operator's args list.
 additionalArgs: []
 
+# -- Array containing extra environment variables which can be templated.
+# For example:
+#  - name: WATCH_NAMESPACE
+#    value: "{{ .Release.Namespace }}"
```

Comment:
Please remove this example. It implies this is somehow officially supported - whereas it's truly not. The core team said they will look into it, but for now it remains unsupported.